cpp/core/shuffle/HashPartitioner.cc

```diff
@@ -43,7 +43,6 @@ arrow::Status gluten::HashPartitioner::compute(
     std::vector<uint32_t>& row2partition,
     std::vector<uint32_t>& partition2RowCount) {
   row2partition.resize(numRows);
-  std::fill(std::begin(partition2RowCount), std::end(partition2RowCount), 0);
```

Comment:
it seems this fill operation is all removed, will this impact the performance?

Reply:
This logic is moved into Hash shuffle writer https://github.com/apache/incubator-gluten/pull/6475/files#diff-0ea9c330c6637a7c2814fd738102f2928bca7b74a99c9f15163a6dce3f3c1e5fR278,  because the `partition2RowCount` computation is not used in sort shuffle writer. I will push some cleanup to completely remove this parameter.